test/AWS.Deploy.CLI.Common.UnitTests/ConfigFileDeployment/DeploymentSettingsHandlerTests.cs

```diff
@@ -0,0 +1,173 @@
+            Assert.Equal(512, GetOptionSettingValue<int>(selectedRecommendation, "TaskMemory"));
+            Assert.Equal("C:\\codebase", GetOptionSettingValue(selectedRecommendation, "DockerExecutionDirectory"));
+        }
+
+        [Fact]
+        public async Task ElasticBeanStalkTests()
+        {
+            // ARRANGE
+            var recommendations = _recommendationEngine.ComputeRecommendations().GetAwaiter().GetResult();
+            var selectedRecommendation = recommendations.FirstOrDefault(x => string.Equals(x.Recipe.Id, "AspNetAppElasticBeanstalkLinux"));
+            var filePath = Path.Combine("ConfigFileDeployment", "TestFiles", "ElasticBeanStalkConfigFile.json");
+
+            // ACT
+            var deploymentSettings = await _deploymentSettingsHandler.ReadSettings(filePath);
+            await _deploymentSettingsHandler.ApplySettings(deploymentSettings, selectedRecommendation, new Mock<IDeployToolValidationContext>().Object);
+
+            // ASSERT
+            Assert.Equal("default", deploymentSettings.AWSProfile);
+            Assert.Equal("us-west-2", deploymentSettings.AWSRegion);
+            Assert.Equal("MyAppStack", deploymentSettings.ApplicationName);
+            Assert.Equal("AspNetAppElasticBeanstalkLinux", deploymentSettings.RecipeId);
+
+            Assert.Equal(true, GetOptionSettingValue(selectedRecommendation, "BeanstalkApplication.CreateNew"));
+            Assert.Equal("MyApplication", GetOptionSettingValue(selectedRecommendation, "BeanstalkApplication.ApplicationName"));
+            Assert.Equal("MyEnvironment", GetOptionSettingValue(selectedRecommendation, "BeanstalkEnvironment.EnvironmentName"));
+            Assert.Equal("MyInstance", GetOptionSettingValue(selectedRecommendation, "InstanceType"));
+            Assert.Equal("SingleInstance", GetOptionSettingValue(selectedRecommendation, "EnvironmentType"));
+            Assert.Equal("application", GetOptionSettingValue(selectedRecommendation, "LoadBalancerType"));
+            Assert.Equal(true, GetOptionSettingValue(selectedRecommendation, "ApplicationIAMRole.CreateNew"));
+            Assert.Equal("MyPlatformArn", GetOptionSettingValue(selectedRecommendation, "ElasticBeanstalkPlatformArn"));
+            Assert.Equal(true, GetOptionSettingValue(selectedRecommendation, "ElasticBeanstalkManagedPlatformUpdates.ManagedActionsEnabled"));
+            Assert.Equal("Mon:12:00", GetOptionSettingValue(selectedRecommendation, "ElasticBeanstalkManagedPlatformUpdates.PreferredStartTime"));
+            Assert.Equal("minor", GetOptionSettingValue(selectedRecommendation, "ElasticBeanstalkManagedPlatformUpdates.UpdateLevel"));
+
+            var envVars = GetOptionSettingValue<Dictionary<string, string>>(selectedRecommendation, "ElasticBeanstalkEnvironmentVariables");
+            Assert.Equal("VarValue", envVars["VarName"]);
+        }
+
+        private static string ResolvePath(string projectName)
```

Comment:
This looks repeated a few times already - is there an easy way to call the one in `SystemIOUtilities`? 

No biggie if it requires changing references (looks like we may only have utilities defined in one of the test projects), can follow-up later.

Reply:
`SystemIOUtilities` did not exist in the `AWS.Deploy.CLI.Common.UnitTests` project. I have added it as a static class so that it can be used by other unit tests in the same namespace.

Reply:
Would prefer it defined once instead of once per project, but don't think this is the only test helper in this situation so okay if we fix up at some point in the future.